Editors/PathEditor.h

```diff
@@ -12,7 +12,7 @@ class PathEditor : public BaseEditor {
 
  public:
   explicit PathEditor(ProtoModelPtr model, QWidget* parent);
-  ~PathEditor();
+  ~PathEditor() override;

```

Comment:
This is a very positive style of change... it'd be nice if we could get clang-tidy checks for stuff like this. Are we building with -Wall?

Reply:
nope.

--   'CMAKE_C_FLAGS_DEBUG': -g
--   'CMAKE_C_FLAGS_MINSIZEREL': -Os -DNDEBUG
--   'CMAKE_C_FLAGS_RELEASE': -O3 -DNDEBUG
--   'CMAKE_C_FLAGS_RELWITHDEBINFO': -O2 -g -DNDEBUG
--   'CMAKE_CXX_FLAGS_DEBUG': -g -fno-omit-frame-pointer -fsanitize=address
--   'CMAKE_CXX_FLAGS_MINSIZEREL': -Os -DNDEBUG
--   'CMAKE_CXX_FLAGS_RELEASE': -O3 -DNDEBUG
--   'CMAKE_CXX_FLAGS_RELWITHDEBINFO': -O2 -g -DNDEBUG